pkg/relayer/session/interface.go

```diff
@@ -0,0 +1,53 @@
+package session
+
+import (
+	"github.com/pokt-network/smt"
+
+	"github.com/pokt-network/poktroll/pkg/observable"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+)
+
+// RelayerSessions is an interface for managing the relayer's sessions and Sparse Merkle Sum Trees (SMSTs).
+// It provides notifications about closing sessions that are ready to be claimed, and handles the creation
+// and retrieval of SMSTs for a given session. It can also be thought of as a SessionManager but dedicated
+// to the Relayer behavior.
+type RelayerSessions interface {
+	// ClosingSessions returns an observable that notifies of sessions ready to be claimed.
+	ClosingSessions() observable.Observable[SessionTree]
```

Comment:
Wdyt about `ClaimableSessions()` and `ProvableSessions()`?

Reply:
Will port this change to #105